lara-typescript/tslint.json

```diff
@@ -19,6 +19,7 @@
     "ordered-imports": false,
     "trailing-comma": false,
     "variable-name": [true, "ban-keywords", "check-format", "allow-pascal-case", "allow-leading-underscore"],
-    "no-namespace": false
+    "no-namespace": false,
+    "max-line-length": false
```

Comment:
I do like having a max line length. It helps to make it easier to compare files side by side. 

But I think I might be the only one that prefers it. :) 
I think @dougmartin didn't really want a max line length.  I can't remember what @kswenson thought about it.

Reply:
I'm 50-50. I usually don't mind max-line-length (although I prefer a bit more relaxed values than 120 ;)). But here we have so many generic types that many lines get pretty long and it's not that easy to divide them nicely. Let me know, I can re-enable it.

Reply:
I don't feel particularly strongly about it. If it's enabled, I wouldn't want it to be any smaller than 120, but having some (reasonably expansive) limit probably isn't a bad idea.